### ISSUE TITLE
cannot remove prior installation of package 'xfun'

### ISSUE THREAD

**Issue by SoyeonLeeBCCUNY** [CrumpLab/rstatsmethods#7]
Some people say you need to close out all projects and restart R.  After restarting R and tried to enter the code, now I got more problems:
package ‘xfun’ successfully unpacked and MD5 sums checked
package ‘digest’ successfully unpacked and MD5 sums checked
Error: Failed to install 'papaja' from GitHub:
  (converted from warning) cannot remove prior installation of package ‘digest’

lesson: you can end up with more problems if you do the above.
update: still cannot install papaja

**Comment by SoyeonLeeBCCUNY** [comment]
Installing papaja
restarted R again.  the error message suggested that "digest" package needs to be installed, so I went into "packages">clicked on "install" and typed in "digest" to install the package.  Then I proceeded with the last line of code to get papaja.  Then here is what happened:
 package ‘xfun’ successfully unpacked and MD5 sums checked
Error: Failed to install 'papaja' from GitHub:
  (converted from warning) cannot remove prior installation of package ‘xfun’
> remove.packages("xfun")
Removing package from ‘C:/Users/AsuS/Documents/R/win-library/4.0’
(as ‘lib’ is unspecified)
Error in remove.packages : there is no package called ‘xfun’

I never had 'xfun'??

Thinking that R wants to remove 'xfun', I installed the package and retried the papaja installation codes again.

**Comment by SoyeonLeeBCCUNY** [comment]
I get through the first three lines of the code okay.  When I get to the last line to "# Install the stable development verions from GitHub" by entering " devtools::install_github("crsh/papaja")
 ", I run into an error message as follows:
package ‘xfun’ successfully unpacked and MD5 sums checked
Error: Failed to install 'papaja' from GitHub:
  (converted from warning) cannot remove prior installation of package ‘xfun’

**Comment by SoyeonLeeBCCUNY** [comment]
This time, I tried "remove.packages("xfun"), then I restarted R.
I tried the code, "devtools::install_github("crsh/papaja")
I then got this:
Error: Failed to install 'papaja' from GitHub:
  (converted from warning) cannot remove prior installation of package ‘rlang’

so this time, I tried removing rlang and then restarted R.
Then R indicated that it wants "rlang" so I installed it again. 

Turning off and then on, tried the code again.  This time, it doesn't want 'glue' . so I repeat the same steps for removing and restaring R.  Repeated the code and now R wants the 'glue' back so I install.  Repeated the steps and this time, I dealt with 'backports' (removing it then, restarting R, then installing, and entering the code to install papaja)

Finally R stopped demanding things and finished installing papaja!
